### ISSUE TITLE
UPBGE issue tracker rules

### ISSUE THREAD

**Issue by lordloki** [UPBGE/upbge#1193]
Hi everybody,

Due to lack of good tools to show/expose in a separate way the bugs, the feature requests, the proposals, etc we are follow the following rules:

- Bugs: It will be tagged as bug or BF-Bender bug. They will be in open status until they are fixed. Adittionally, a label with 0.2.5 (old render) or 0.3 (new eevee render) will be added to triage better. Also, If the bug is a crash a "crash" tag will be added.

- Feature request or proposals: They will be tagged as "Feature request for later" / "Proposal for later" and "open". After tagged they will be closed. This closing doesn't mean that we discard the feature but we need the tracker clean. When we want to expose that we discard/implement the feature we will remove the "open"tag and We will add a "discard" or "implemented".

Basically is to give preference to bugs and to have a crear vision of the status of the master/branchs.

Thank you for your understanding
